.github/ISSUE_TEMPLATE/2_bug-or-performance-issue-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug or performance issue report
+about: Create a report to help us improve oneDPL
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug or performance issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```

Comment:
Please change this to "The steps to reproduce the behavior."

Reply:
Done.